src/main/resources/help/content/releasenotes.md

```diff
@@ -1,3 +1,9 @@
+#### Version 9.4.1
+
+##### Documentation updates
+
+* Added to the documentation the [organize components by layer](running.md#organizing-components-by-layer) feature (properties bdio.organize.components.by.layer and bdio.include.removed.components).
```

Comment:
Do we not want to document this as a feature because some customers have been instructed to use the property while it was private?  I could see framing it as a feature in that we're publicizing a private API